### PR TITLE
expand scope of `moderate` to include threads. Allow for locking

### PR DESCRIPTION
**Pull request by MylesBorins**
8e05be9 (Myles Borins, 50 seconds ago)
   moderation: expand scope to threads

   The current definition of moderation is scoped to a specific post not and
   entire thread

a66c539 (Myles Borins, 2 minutes ago)
   moderation: be explicit about locking

   Moderation didn't currently cover the scope of locking a thread.

### CODE REVIEW

File: Moderation-Policy.md [hunk 1]
```diff
@@ -54,9 +54,9 @@ this default policy applies.
 * *CommComm* refers to the [Node.js Community Committee][].
 * *Post* refers to the content and titles of any issue, pull request, comment
   or wiki page.
-* *Moderate* refers to the act of modifying the content and title of, or
-  deleting, any Post for the purpose of correcting or addressing Code of Conduct
-  violations.
+* *Moderate* refers to the act of modifying the content and title of, locking or
+  deleting, any Post or thread for the purpose of correcting or addressing Code
```
Comment (Trott) [inline Moderation-Policy.md]: What's a thread in this context? (If a "thread" is made up of posts, I don't think we need to specify it.)

Comment (MylesBorins) [reply Moderation-Policy.md]: A thread is made up of posts. it is a PR or an ISSUE

**Comment by MylesBorins** [comment]
The main concern was with regards to deleting and locking

You can't lock a post

**Comment by Trott** [comment]
> You can't lock a post

You can lock issues and PRs. I'd use those terms rather than threads.

**Comment by Trott** [comment]
(Another possibility is to mirror GitHub's terminology. The interface says "Lock conversation" and I think saying "locking conversations" is pretty self-explanatory, at least more so than "locking threads".)

**Comment by MylesBorins** [comment]
Just noticing this hasn't landed. Quick ping for new comments or objections, will land Monday if there are none

**Comment by Trott** [comment]
Since _thread_ is non-standard and subject to interpretation, please remove it. Use the same terminology GitHub does for clarity: post, comment, issue, pull request, or conversation as appropriate.

**Comment by Trott** [comment]
I’ll try to go through later with ```suggestion blocks when I’m not on a phone.

File: Moderation-Policy.md (hunk 1 above)
Comment (Trott) [reply Moderation-Policy.md]: I pushed a commit with my suggestion to improve this definition. Force-push it out if you hate it. Thanks.

**Comment by MylesBorins** [comment]
this has been open for a while with no objections... going to land

**Pull request merged**